packages/app-admin/src/components/AppInstaller/styled.tsx

```diff
@@ -39,11 +39,12 @@ export const InstallContent = styled("div")({
 });
 
 export const installerSplitView = css({
+    // height: "100vh",
     ".webiny-split-view__inner": {
-        height: "100vh",
-        ".webiny-split-view__right-panel-wrapper": {
-            height: "100vh"
-        }
+        height: "100vh"
+    },
+    ".webiny-split-view__right-panel-wrapper": {
+        height: "100vh"
     }
 });
```

Comment:
If I remember this correctly, this `100vh` should apply to both left and right panels. With this new rule it looks like you're only applying that height to the right panel.

Reply:
This is just a syntax change, I've removed the nesting from the declaration as it wasn't getting evaluated.